web/js/containers/sidebar/sidebar.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
```

Comment:
Event icons are not removed from the map in this workflow:

1. Fresh page, select Events Tab, let load
2. Select Data Tab
3. Select Events Tab
3. Select Layers Tab

Reply:
Got a fix for this incoming with next push 👍🏻